hdtv/database/PGAALibraries.py

```diff
@@ -128,18 +133,18 @@ def open(self):
                 except ValueError:
                     halflife = None
                 gamma = PGAAGamma(
-                    Nuclides(
-                        Z,
-                        A)[0],
+                    Nuclides(Z, A)[0],
```

Comment:
Codacy found an issue: ['Nuclides' may be undefined, or defined from star imports: hdtv.database.common (F405)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)